org.qutebrowser.qutebrowser.yml

```diff
@@ -0,0 +1,40 @@
+---
+app-id: org.qutebrowser.qutebrowser
+runtime: org.kde.Platform
+runtime-version: '5.12'
+sdk: org.kde.Sdk
+command: qutebrowser
+rename-desktop-file: qutebrowser.desktop
+rename-icon: qutebrowser
+finish-args:
+- "--share=ipc"
+- "--socket=x11"
+- "--socket=wayland"
+- "--socket=pulseaudio"
+- "--share=network"
+- "--filesystem=xdg-download"
+- "--device=dri"
+cleanup:
+- "/lib/debug"
+modules:
+- pygments.yml
+- pypeg2.yml
+- markupsafe.yml
+- jinja2.yml
+- attrs.yml
+- pyyaml.yml
+- sip.yml
```

Comment:
The manually written modules should be inlined, it just makes it a pain to navigate the manifest otherwise. For the other stuff, which I guess you used [flatpak-builder-tools](https://github.com/flatpak/flatpak-builder-tools/) to generate (I hope – if you did not, please do), keep them in the generated form, i.e. with the `python3-` prefix and JSON formatting – converting them to YAML and renaming just adds additional work when updating them later.